src/mux/comment.ts

```diff
@@ -0,0 +1,185 @@
+import Mux, { IMuxRequest, IRequestData } from '../framework/mux';
+import { IResponseList, IOrdering, IResponseRecord } from '../framework/response';
+import { ModelArticle } from '../model/article';
+import { IComment, ModelComment } from '../model/comment';
+import { ModelUser } from '../model/user';
+import { ArticleUuidValidator, CommentUuidValidator, CommentValidator } from '../validators';
+
+// Get all comments at root level in article for a given article's UUID
+Mux.get<IComment>(
+  '/v1/comment/article/:articleUuid',
+  ArticleUuidValidator,
+  async (requestData: IRequestData): Promise<IResponseList<IComment>> => {
+    const offset = requestData.body.offset > 1 ? requestData.body.offset - 1 : 0;
+    const limit = requestData.body.limit ? requestData.body.limit : 10;
```

Comment:
Should have a maximum value for body limit.

Btw, you shouldn't access value directly and pagination field need to be verified too.